ChangeLog

```diff
@@ -18,6 +18,9 @@ Bugfix
      return value. Found by @davidwu2000. #839
    * Fix a memory leak in mbedtls_x509_csr_parse(), found by catenacyber,
      Philippe Antoine. Fixes #1623.
+   * Remove wrong documentation for `mbedtls_ssl_get_session`.
```

Comment:
In english, you wouldn't say 'wrong documentation' but 'incorrect documentation'. The wrong documentation means a different document unrelated to the subject, or something.

Can I suggest we make this a positive statement to correct this?

```
  * Correct the documentation for `mbedtls_ssl_get_session()`.
    This API has deep copy of the session, and the peer
    certificate is not lost. #926
```